opm/simulators/linalg/ISTLSolverBda.cpp

```diff
@@ -43,6 +43,10 @@
 
 #include <opm/grid/polyhedralgrid.hh>
 
+#include <thread>
+
+std::shared_ptr<std::thread> copyThread;
```

Comment:
Isn't there a way without the global variable, e.g. passing it around to the method where it is joined?

Reply:
Yes that is possible, but that would mean to pass it down via solve_system method, so modifying the solvers interface by adding something that is not really related to the linear system data, which I like even less than the global variable. But if this is a strong objection, I modify the code to pass it around, unless there are also other options that I am not aware of.